GRDB/Core/Support/Foundation/UUID.swift

```diff
@@ -17,7 +21,17 @@ extension NSUUID : DatabaseValueConvertible {
         guard let data = NSData.fromDatabaseValue(databaseValue), data.length == 16 else {
             return nil
         }
+        #if os(Linux)
```

Comment:
Maybe the `constructing an object of class type 'Self' with a metatype value must use a 'required' initializer` error can be addressed with the `cast()` function defined in [Utils.swift](https://github.com/groue/GRDB.swift/blob/v0.106.1/GRDB/Core/Utils.swift#L43-L46). See https://github.com/groue/GRDB.swift/blob/v0.106.1/GRDB/Core/Support/Foundation/NSData.swift#L17 for an example